### ISSUE TITLE
Create a separate note for optimized publications

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2802]
### Section

https://www.w3.org/TR/epub-a11y-111/#sec-optimized-pubs

### Describe the problem

Optimized publications aren't in the scope of the epub accessibility document but yet we have a whole informative section on them.

I think we added the section at least in part to avoid any offence that republishers targeting specific user groups are making inaccessible content because they don't conform to the standard, but we could do that without making it a section of a normative document.

### Describe the fix or new feature you propose

Should we consider moving this section to its own note and reference it from the accessibility specification?

We could also choose to make it an appendix to get it out of the body where it doesn't belong, but that still leaves it as an informative aside in a normative document.

**Comment by iherman** [comment]
Personally, I would opt for an appendix in the A11y document. The whole section seems to be way too short to be a WG Note.

An additional observation: the Note after the core text (before the examples) refers to 

https://w3c.github.io/epub-specs/docs/optimized-pubs/

but that page does not provide a "non-normative list of standards" as it says in the note and the page itself does provide much information (and it refers back to this document, creating a funny reference circle). The only document it refers to is the Daisy one:

http://www.daisy.org/guidelines/epub/navigable-audio-only-epub3-guidelines

 Or is it the idea that the group will provide a full list of such document? Because if not, the note should refer to the Daisy document instead, with an appropriate content reference (instead of the list of standards).

**Comment by mattgarrish** [comment]
> Personally, I would opt for an appendix in the A11y document. The whole section seems to be way too short to be a WG Note.

End of the day, either works for me. It's just that, unlike the informative distribution section, optimizing publications for a particular user need doesn't directly influence what you need to do to create universally accessible content.

It's a relic from IDPF 1.0 when we actually defined optimized publications as [a formal conformance class](https://idpf.org/epub/a11y/#sec-conf-opt). In the ISO 1.0 process it was rightly noted that this is out of scope but we left the guidance by making it non-normative. It should have been removed from the body, too.

> The only document it refers to is the Daisy one:

That's the only one the group was aware of at the time. The document was originally an [informative IDPF registry](https://idpf.github.io/epub-registries/optimizations/). Maybe ebraille could be listed there, too, but there aren't a lot of formal standards for creating optimized epubs and probably won't be.

That's the problem with these kinds of registries. They get forgotten once we publish them and no one seeks them out to list their guidelines unless it's a participating organization in the WG.

**Comment by mattgarrish** [comment]
> The whole section seems to be way too short to be a WG Note.

What about a CG note? This isn't technology that the WG is going to develop.

I could also see dropping the list of optimization standards as a separate document if this isn't under the WG.

**Comment by iherman** [comment]
> > The whole section seems to be way too short to be a WG Note.
> 
> What about a CG note? This isn't technology that the WG is going to develop.
> 
> I could also see dropping the list of optimization standards as a separate document if this isn't under the WG.

Having the "registry" (though containing on reference only for now) managed by the CG probably makes sense; a CG has more freedom to organize its publications as it wishes. I still see some value in the text currently in the spec, i.e., keeping in the document (as an appendix) looks o.k. to me.

Something the A11y task force could discuss? @avneeshsingh ?
